tests/cli-self-upd.rs

```diff
@@ -302,219 +302,9 @@ fn uninstall_doesnt_leave_gc_file() {
-fn uninstall_removes_path_from_profile() {
-    uninstall_removes_path_from_rc(".profile");
-}
-
-#[test]
-#[cfg(unix)]
-fn uninstall_removes_path_from_bash_profile() {
-    uninstall_removes_path_from_rc(".bash_profile");
-}
-
-#[test]
-#[cfg(unix)]
-fn uninstall_doesnt_touch_rc_files_that_dont_contain_cargo_home() {
-    setup(&|config| {
-        let my_rc = "foo\nbar\nbaz";
-        expect_ok(config, &["rustup-init", "-y"]);
-        expect_ok(config, &["rustup", "self", "uninstall", "-y"]);
-
-        let profile = config.homedir.join(".profile");
-        raw::write_file(&profile, my_rc).unwrap();
-
-        let profile = fs::read_to_string(&profile).unwrap();
-
-        assert_eq!(profile, my_rc);
-    });
-}
-
-// In the default case we want to write $HOME/.cargo/bin as the path,
-// not the full path.
-#[test]
-#[cfg(unix)]
-fn when_cargo_home_is_the_default_write_path_specially() {
-    setup(&|config| {
-        // Override the test harness so that cargo home looks like
-        // $HOME/.cargo by removing CARGO_HOME from the environment,
-        // otherwise the literal path will be written to the file.
-
-        let my_profile = "foo\nbar\nbaz";
-        let profile = config.homedir.join(".profile");
-        raw::write_file(&profile, my_profile).unwrap();
-        let mut cmd = clitools::cmd(config, "rustup-init", &["-y"]);
-        cmd.env_remove("CARGO_HOME");
-        assert!(cmd.output().unwrap().status.success());
-
-        let new_profile = fs::read_to_string(&profile).unwrap();
-        let expected = format!("{}\nexport PATH=\"$HOME/.cargo/bin:$PATH\"\n", my_profile);
-        assert_eq!(new_profile, expected);
-
-        let mut cmd = clitools::cmd(config, "rustup", &["self", "uninstall", "-y"]);
-        cmd.env_remove("CARGO_HOME");
-        assert!(cmd.output().unwrap().status.success());
-
-        let new_profile = fs::read_to_string(&profile).unwrap();
-        assert_eq!(new_profile, my_profile);
-    });
-}
-
-#[test]
-#[cfg(windows)]
-/// Smoke test for end-to-end code connectivity of the installer path mgmt on windows.
-fn install_uninstall_affect_path() {
-    setup(&|config| {
-        let path = config.cargodir.join("bin").to_string_lossy().to_string();
-
-        expect_ok(config, &["rustup-init", "-y"]);
-        assert!(
-            get_path().unwrap().contains(&path),
-            format!("`{}` not in `{}`", get_path().unwrap(), &path)
-        );
-
-        expect_ok(config, &["rustup", "self", "uninstall", "-y"]);
-        assert!(!get_path().unwrap().contains(&path));
-    });
-}
-
-#[test]
-#[cfg(unix)]
-/// This covers both windows and unix no-path-modification cases due to the code structure.
-fn install_doesnt_modify_path_if_passed_no_modify_path() {
-    setup(&|config| {
-        let profile = config.homedir.join(".profile");
-        expect_ok(config, &["rustup-init", "-y", "--no-modify-path"]);
-        assert!(!profile.exists());
-    });
-}
+#[ignore]
+fn uninstall_stress_test() {}
```

Comment:
Why adding this?

Reply:
https://tahoe-lafs.org/~zooko/badmerge/simple.html

Reply:
git rebase is drunk? I didn't intentionally make that edit, but I have had to rebase this several times and some oddities have arisen at this point (git has no concept of changes, it only has diffs between trees).